### DISCUSSION TITLE
Cable for the photodiode?

### DISCUSSION THREAD

**Discussion by tokehgekko** [klaasnicolaas/home-assistant-glow#292] [Q&A]
Would it be possible to add a long-ish cable for the photodiode itself to be able to keep everything indoors?
I live where it gets really cold in the winter.
Would this have any bad effects?

**Comment by slepax** [comment]
How long would the cable be? I've actually separated the photodiode from the main circuit with about 50cm of wire.

If you want to go quite far I would leave the photodiode with the breakout board together and run a long wire to the ESP inside your home. This way you will have only the digital signal transmitted over the long[er] wire.

**Comment by klaasnicolaas** [comment]
In the past I have extended the photodiode by about half a meter, which did not cause any problems for me. But I don't know how many meters you want to bridge exactly?
